extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -150,29 +152,73 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					// go back to select setting quickpick if user escapes from inputting the setting name in case they changed their mind
 					continue;
 				}
-
-				const newConnectionStringValue = await vscode.window.showInputBox(
+				let connectionStringOptions: vscode.QuickPickItem[] = [
 					{
-						title: constants.enterConnectionString,
-						ignoreFocusOut: true,
-						value: 'Server=localhost;Initial Catalog={db_name};User ID=sa;Password={your_password};Persist Security Info=False',
-						validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
+						label: constants.connectionProfile
```

Comment:
If it's just a label you don't need objects, you can pass in an array of strings directly.

Also the var name `connectionStringOptions` could be updated I think - maybe something like `connectionStringSelectionOptions` or something? (and then `selectedConnectionStringOption`) Not a huge deal if you can't think of a better one though.